slaves/linux-cross/Dockerfile

```diff
@@ -38,6 +36,30 @@ RUN pip install buildbot-slave
 RUN groupadd -r rustbuild && useradd -r -g rustbuild rustbuild
 RUN mkdir /buildslave && chown rustbuild:rustbuild /buildslave
 
+# Install arm cross compiler
+WORKDIR /build
+COPY linux-cross/build_arm_toolchain_root.sh /build/
+RUN /bin/bash build_arm_toolchain_root.sh && chown rustbuild:rustbuild /build
+RUN mkdir /x-tools && chown rustbuild:rustbuild /x-tools
```

Comment:
There's a catch here's that I forgot to mention. crosstool-ng can't be run as root, so we run it under the rustbuild user. That's the reason `/x-tools` directory needs to be writable by rustbuild.